.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
   "eslint.lintTask.options": "--config ./tools/lint/eslint.config.js",
   "eslint.format.enable": true,
   "eslint.runtime": "node",
-  "eslint.workingDirectories": ["./"]
+  "eslint.workingDirectories": ["./"],
+  "typescript.preferences.importModuleSpecifier": "project-relative"
```

Comment:
adding this after finding this SO: https://stackoverflow.com/questions/60200359/tell-vs-code-to-always-use-relative-paths-for-typescript-auto-imports

it's been working well for me so far. I figure we can remove it if it creates unexpected auto imports for anyone